eng/Version.Details.xml

```diff
@@ -412,17 +412,17 @@
       <Uri>https://github.com/dotnet/extensions</Uri>
       <Sha>275e691f7e575f208290d1cbb8cb450f4a3a85d6</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.GenAPI" Version="5.0.0-beta.20076.3">
+    <Dependency Name="Microsoft.DotNet.GenAPI" Version="5.0.0-beta.20074.6">
```

Comment:
At least this version also appears in eng/Versions.props. Needs to be fixed there too.

Reply:
Should have said: My approval is conditional on fixing this.